src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -207,16 +220,20 @@ module.exports = function<T, P, I, TI, C>(config : HostConfig<T, P, I, TI, C>) {
     // already been invoked.
     effectfulFiber = finishedWork.firstEffect;
     while (effectfulFiber) {
+      const current = effectfulFiber.alternate;
+
+      // Use Task priority for lifecycle updates
+      const previousPriorityContext = priorityContext;
+      priorityContext = TaskPriority;
       if (effectfulFiber.effectTag & (Update | Callback)) {
-        const current = effectfulFiber.alternate;
-        const previousPriorityContext = priorityContext;
-        priorityContext = TaskPriority;
-        try {
-          commitLifeCycles(current, effectfulFiber);
-        } finally {
-          priorityContext = previousPriorityContext;
-        }
```

Comment:
Removing this try/finally block seems responsible for some of the test failures.
If I add it back I get:

```diff
--- a/scripts/fiber/tests-passing.txt
+++ b/scripts/fiber/tests-passing.txt
@@ -470,7 +470,6 @@ src/renderers/dom/__tests__/ReactDOMProduction-test.js
 * should use prod React
 * should handle a simple flow
 * should call lifecycle methods
-* should throw with an error code in production
 
 src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js
 * should render strings as children
@@ -1021,8 +1020,12 @@ src/renderers/shared/shared/event/eventPlugins/__tests__/ResponderEventPlugin-te
 
 src/renderers/shared/stack/reconciler/__tests__/ReactComponent-test.js
 * should throw when supplying a ref outside of render method
-* should support refs on owned components
+* should warn when children are mutated during update
 * should not have refs on unmounted components
+* should support new-style refs
+* should support new-style refs with mixed-up owners
+* should call refs at the correct time
+* fires the callback after a component is rendered
 
 src/renderers/shared/stack/reconciler/__tests__/ReactComponentLifeCycle-test.js
 * should not reuse an instance when it has been unmounted
@@ -1185,8 +1188,9 @@ src/renderers/shared/stack/reconciler/__tests__/ReactStatelessComponent-test.js
 * should update stateless component
 * should unmount stateless component
 * should pass context thru stateless component
-* should provide a null ref
+* should use correct name in key warning
 * should support default props and prop types
+* should receive context
 * should work with arrow functions
 * should allow simple functions to return null
 * should allow simple functions to return false
```